packages/grid/x-data-grid/src/components/ErrorOverlay.tsx

```diff
@@ -16,9 +26,9 @@ export const ErrorOverlay = React.forwardRef<HTMLDivElement, ErrorOverlayProps>(
     const defaultLabel = apiRef.current.getLocaleText('errorOverlayDefaultLabel');
 
     return (
-      <GridOverlay ref={ref} {...other}>
+      <ErrorGridOverlay ref={ref} {...other}>
```

Comment:
`GridOverlay` is a styled component. You could use `sx`. You may need to fix some TS types to accept this prop.

Reply:
Nice